### PR TITLE
[DOC] Add validator domain verification setup instructions

### PR DESCRIPTION
**Pull request by wilsonianb**
DEC-291

### CODE REVIEW

File: content/rippled-setup.md [hunk 1]
```diff
@@ -236,4 +236,48 @@ Becoming a validator that participates in the network involves several steps. In
...
+              "master_seed_hex" : "434256443542C27BD1A84A2BACC9B8F0",
+              "public_key" : "aBQzwnRdgHVZmr8gLNugihTf5NsWAUpayGdAHtz8YPk1w3L4fh6S",
+              "public_key_hex" : "038ED9785EE7FC687445E0D94065A74FF6CEC6506A03C7380075D81A2B9E7E8681",
+              "status" : "success"
+           }
+        }
+
+2. Fund the account by sending it at least 25 XRP
+
+3. Set the account domain field to the your domain with ripple.tx
```
Comment (mDuo13) [inline content/rippled-setup.md]: I think this is supposed to be ripple.txt, not ripple.tx?

File: content/rippled-setup.md [hunk 2]
```diff
@@ -236,4 +236,48 @@ Becoming a validator that participates in the network involves several steps. In
...
+
+### ripple.txt
+
+Publish a [ripple.txt](https://wiki.ripple.com/Ripple.txt) page at your domain with a signed SSL certificate.
+
+List the validator's `validation_public_key` (generated [above](#validator-setup) in step 3) in the `[validation_public_key]` section.
+
+### Account domain
+
+1. Get the validator's account id using the `validation_seed` generated [above](#validator-setup) in step 3:
```
Comment (mDuo13) [inline content/rippled-setup.md:255]: we probably want to add more text describing what a validator's account ID is.

Comment (wilsonianb) [inline content/rippled-setup.md:255]: Added text

File: content/rippled-setup.md (hunk 1 above)
Comment (wilsonianb) [inline content/rippled-setup.md]: fixed

**Comment by mDuo13** [comment]
I have deployed the changes to production (with a few more typo fixes). Sorry for taking so long to get it out.

**Comment by wilsonianb** [comment]
No problem. Thanks for the review and merge.

**Pull request merged**